File: class12/cpp/AP1201-AnoBissexto.cpp
#include <iostream>
int main() {
    int ano;
    std::cout << "Digite o ano: ";
    std::cin >> ano;
    bool bissexto = (ano % 4 == 0 && ano % 100 != 0) || (ano % 400 == 0);
    std::cout << ano << (bissexto ? " E bissexto" : " NAO e bissexto") << std::endl;
    return 0;
}

File: class12/python/AP1201-AnoBissexto.py
# Verifica se um ano e bissexto
ano = int(input("Digite o ano: "))
bissexto = (ano % 4 == 0 and ano % 100 != 0) or (ano % 400 == 0)
print(f"{ano} {'E' if bissexto else 'NAO e'} bissexto")
